client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -39,6 +39,24 @@ export const useRequestColumns = () => {
         getSortValue: rowData => rowData.item.name,
       },
     ],
+    {
+      key: 'unit',
+      label: 'label.unit',
+      align: ColumnAlign.Left,
+
+      // width: 100,
+      accessor: ({ rowData }) => rowData.item.unitName,
+      getSortValue: rowData => rowData.item.unitName ?? '',
+    },
+    {
+      key: 'defaultPackSize',
+      label: 'label.default-pack-size',
```

Comment:
So there is already a "label.default-pack-size" string in "catalogue.json", but it seems like the table columns only use "common". Rather than changing the inner workings of the Table renderer, I just made a new string in common. That okay?

Reply:
yep. can you remove the one in `catalogue` then? It isn't required

Reply:
The `catalogue` one is actually used in Item -> Detail View -> Tabs -> General.tsx.

Reply:
yeah, but if you add it to `common` then the General.tsx will be able to get it from there; common is the fallback file.